apps/mobile/src/store/key-store.ts

```diff
@@ -109,6 +127,60 @@ export function useKeyStore() {
       const nextAccountIndex =
         fingerprintAccounts.length === 0 ? 0 : highestKeychainAccountIndex + 1;
 
+      // in extension secretKey is the mnemonic
+      const secretKey = mnemonic;
+
+      const legacyAccountActivityLookupGaia =
+        await checkForLegacyGaiaConfigWithKnownGeneratedAccountIndex(secretKey);
+
+      // hardcoding this to match extension count for dev wallet
+      // checkForLegacyGaiaConfigWithKnownGeneratedAccountIndex fails with
+      // [TypeError: _this.nodePbkdf2 is not a function (it is undefined)]
+      const legacyAccountActivityLookup =
+        legacyAccountActivityLookupGaia === 0 ? 123 : legacyAccountActivityLookupGaia;
```

Comment:
suggest we don't add any of the gaia checks here, resolving the import issue, check stacks/bitcoin balances only